src/etapi/spec.ts

```diff
@@ -1,10 +1,12 @@
-const fs = require('fs');
-const path = require('path');
+import { Router } from "express";
+
+import fs = require('fs');
+import path = require('path');
 
 const specPath = path.join(__dirname, 'etapi.openapi.yaml');
-let spec = null;
+let spec: any = null;
```

Comment:
Small improvement: Use `string | Buffer | null` instead of `any`

Reply:
@alexpietsch , good catch. Seems that `string | null` is sufficient, since the encoding is provided to `readFileSync`.